include/cvc5/cvc5_skolem_id.h

```diff
@@ -175,7 +175,8 @@ enum ENUM(SkolemId) : uint32_t
    */
   EVALUE(SHARED_SELECTOR),
   /**
-   * The n^th skolem for quantified formula Q.
+   * The n^th skolem for quantified formula Q, where Q is a universally quantified
+   * formula.
```

Comment:
```suggestion
   * The n^th skolem for the negation of universally quantified formula Q.
```
Note: this caught me off guard. From the old documentation, I assumed `Q` to be either an existentially quantified formula or the negation of a universally quantified one.